File: Source/prx/Source/Library/PrxSample/prx/src/main/java/com/philips/cdp/prxclient/datamodels/specification/FeatureItem.java
package com.philips.cdp.prxclient.datamodels.specification;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

import java.io.Serializable;

public class FeatureItem implements Serializable {

	private static final long serialVersionUID = -4070255334532209384L;
	@SerializedName("code")
	@Expose
	public String code;

	@SerializedName("rank")
	@Expose
	public String rank;

	@SerializedName("referenceName")
	@Expose
	public String referenceName;
}

File: Source/prx/Source/Library/PrxSample/prx/src/main/java/com/philips/cdp/prxclient/datamodels/specification/FeatureItem.kt
package com.philips.cdp.prxclient.datamodels.specification

import android.os.Parcelable
import kotlinx.android.parcel.Parcelize

@Parcelize
data class FeatureItem(var code: String? = null, var rank: String? = null, var referenceName: String? = null) : Parcelable